targets/TARGET_STM/TARGET_STM32L4/TARGET_MTS_DRAGONFLY_L471QG/ONBOARD_SARA4_PPP.cpp.autosave

```diff
@@ -0,0 +1,132 @@
+/* mbed Microcontroller Library
```

Comment:
I guess this file got added by mistake:
targets/TARGET_STM/TARGET_STM32L4/TARGET_MTS_DRAGONFLY_L471QG/ONBOARD_SARA4_PPP.cpp.autosave

Reply:
Thanks Kimmo, removed